### DISCUSSION TITLE
Elastic Licensing

### DISCUSSION THREAD

**Discussion by kvkamin** [Security-Onion-Solutions/securityonion#11242] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

20

### RAM

64gb

### Storage for /

320GB

### Storage for /nsm

8TB

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Apparently the coolest features in Elastic Defend requires a license at the Premium level.  I filled out the sales form on Elastic Search web several days ago and have not received a response.  Has anyone successfully purchased a license?  How is it priced?

K 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment, marked as answer]
If you still haven't heard back from Elastic, you can reach out to our sales team using the Contact Us button at https://securityonionsolutions.com/ and they can talk with you.
